opennms-doc/guide-development/src/asciidoc/text/rest/measurements.adoc

```diff
@@ -16,8 +16,9 @@ The following table shows all supported query string parameters and their defaul
 [options="header"]
 |===
 | name               | default   | comment
-| start              | -14400000 | Timestamp in milliseconds. If < 0, the effective value will be (end + start).
-| end                | 0         | Timestamp in milliseconds. If <= 0, the effective value will be the current timestamp.
+| start              | -14400000 | Timestamp in milliseconds. If < 0, the effective value will be `end - start`
```

Comment:
`end + start` is definitely correct. The reason for this is, that start can be negative. If it were `-` a negative value would result in `end - -start = end + start`. This seems wrong for the default value of `-14400000`.